agent/node.go

```diff
@@ -49,6 +49,10 @@ type NodeConfig struct {
 	// Secret to be used on the first certificate request.
 	Secret string
 
+	// ExternalCAURLs is a list of initial URLs to which a manager node
+	// will make certificate signing requests for node certificates.
+	ExternalCAURLs []string
```

Comment:
Have we confounded node and cluster config? We should figure out how to separate these...